flake.nix

```diff
@@ -0,0 +1,39 @@
+{
+  # Override nixpkgs to use the latest set of node packages
+  inputs.nixpkgs.url = "github:NixOS/nixpkgs/master";
+  inputs.systems.url = "github:nix-systems/default";
+
+  outputs =
+    {
+      self,
+      nixpkgs,
+      flake-utils,
+      systems,
+    }:
+    flake-utils.lib.eachSystem (import systems) (
+      system:
+      let
+        pkgs = import nixpkgs { inherit system; };
+      in
+      {
+        devShells.default = pkgs.mkShell {
+          buildInputs = with pkgs; [
+            # generic
+            git
+            git-lfs
+            gnumake
+            gnutar
+
+            # frontend
+            nodejs
+            nodePackages.npm
```

Comment:
Since `npm` is bundled in the `nodejs` package I don't think it's necessary to have here. 
Further, do we want to lock which version of the `nodejs` package we're using here? (Same question for `go` down below). 
Probably not an issue right now, but could be as we update the nixpkgs input later.